### PR TITLE
chore: prep 0.30.0 release

### PR DESCRIPTION
**Pull request by Panaetius**
needed for https://github.com/SwissDataScienceCenter/renku-python/pull/3178

/deploy #persist extra-values=global.renku.cli_version=2.6.0rc1

### CODE REVIEW

File: helm-chart/renku/requirements.yaml [hunk 1]
```diff
@@ -13,7 +16,7 @@ dependencies:
   version: 1.15.2
 - name: renku-gateway
   alias: gateway
-  repository: "https://swissdatasciencecenter.github.io/helm-charts/"
+  repository: "https://swissdatasciencecenter.github.io/helm-charts/" #"file:///home/zenon/DEV/ETH/renku-gateway/helm-chart/renku-gateway"
```
Comment (olevski) [inline helm-chart/renku/requirements.yaml:19]:
```suggestion
  repository: "https://swissdatasciencecenter.github.io/helm-charts/"
```

File: ralf.values.yaml
Comment (olevski) [inline ralf.values.yaml:1]: Reminder to remove this from the PR.

File: helm-chart/renku/requirements.yaml [hunk 2]
```diff
@@ -1,8 +1,11 @@
 dependencies:
 - name: renku-core
   alias: core
-  repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 2.3.2
+  repository: "file:///home/zenon/DEV/ETH/renku-python/helm-chart/renku-core"
+  version: 2.4.0
+  import-values:
+  - child: versions
+    parent: global.core.versions
```
Comment (olevski) [inline helm-chart/renku/requirements.yaml:8]: So the only reason for this is to then use `global.core.versions` in the renku-gateway helm chart?

**Review by olevski — CHANGES_REQUESTED** [review]

**Review by olevski — APPROVED** [review]

**Review by aledegano — DISMISSED** [review]

**Review by aledegano — DISMISSED** [review]

**Review by olevski — APPROVED** [review]

**Review by bethcg — COMMENTED** [review]
Looks good !

**Pull request merged**